### PR TITLE
Bump primitive >= 0.6.4.0

### PR DESCRIPTION
**Pull request by Avi-D-coder**
0.7 removed `Addr`

### CODE REVIEW

File: tests/Smoke.hs [hunk 1]
```diff
@@ -172,8 +172,7 @@ testContention (newChan,writeChan,readChan,_) writers readers n = do
   mapM_ (forkCatching False "testContention writeChan i " . mapM_ (writeChan i)) groups
 
   ns <- replicateM nNice (C.readChan out)
-  isEmpty <- C.isEmptyChan out
-  if sort ns == [1..nNice] && isEmpty
```
Comment (Avi-D-coder) [inline tests/Smoke.hs]: I had to remove `isEmpty` because it is no long in base. Does this mess up the test?

**Comment by Avi-D-coder** [comment]
I get this test error with my changes, but testing without keeps freezing blocked before deadlock tests.
If you can test this on your end that would be great.

```
testing Unagi.Bounded:
    Checking for deadlocks from killed reader, x10000... .*...*..*...*...*...*...*...*...*...*...*...*...*...*...*...**..*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*.......*...*.......*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*.......*...*.*.*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*....*...*...*...*...*...*...**..*...*...**.*...*...*...*...*.*.*...*...*...*...*...*...*...*...*..*...***.*.*.*...*...*...*.***...*...*...*.......*.......*...*...*...*...*...*..*...*...*...*...*...*...*...*...*.......*...*...*...*...*...*...*...*...*...*.*.**..**..*...*...*...***.***.*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...**..*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...**..*...*..****.*...*.*.*...*...*..**.*.*...*...*...**..*.*.*...*...*.*.*...*...*...*...*...*...*...*...*...*...*...*...***...*...*...*...*...*.**...*...**..*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...**..*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*.......*...*...*...*...*...*...*...*...*...*...*...*.......*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*..*...*...*...*...*.*.*...*...*...*...*...*.*.*..**...**..*...**..*.****..*...*...*...**..*...*...*...*...*...*...**..***.*...*...*...*...*...**..*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*.*.*...*...*...*...*...*...*...*.*...*...**..*.......*..**..*...*...*...*...*...*...*...*...*...*...*.**..*...*...*...*...*..**...*...*...*.**...*..***..*...*...**..**..*...*...*...*...*...*...**..*...*...*...*...*...*...*...*...*...*..**...*.......*...*...*...*...*...*...*...*.*.*...*...*...*...*...*...*...*...**.*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*.......*...*...*...*...*...*...*...*...*...*...*...*...*.*.**..**..*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...**..*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*..*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*.......*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*.......*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*.......*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...**...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...**..*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*..**...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*.......*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*.......*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*.......*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*.*.*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*..*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*..**...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*.......*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*..*...*...*...*...*...*...*.......*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*.......*...*...*...*...*...*...*...*.......*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*.......*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...***.....*...*...*...*...*...*...*...*...*...*...*...*...*...*...*..**...*...*...*...*...*...*...*...*...*...*...**..*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...**..*...*...*...*..*...*...**..*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...**..*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*.*.*...*...*...*...*...*...*...*...*...*...*...*...*..**...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*..**...*...*...*...*...*...*...*.......*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*.......*...*...*...*.......*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*.*.*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*.......*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*.*.*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*.......*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*.*.*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*.......*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...**..*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...**..*...*...*...*...*...*...*...*...*.......*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*.......*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...**..*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*..
.*...*...*...*...*...*...*...*...*.......*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...**..*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...**..*...*...*...*...*...*...*.*.*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...**..*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*.......*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...**..*...*...*...*...*...*...*...*...*...*...*...*...*...*.......*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*..*...*...*...*...*...*...**..*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*.......*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*..*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*..*...*...*...*...*...*.*.*...*...*...*...*...*.......*...*...*...*...*..*.*.*...*...*...*...*...*...*.......*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*..*...*...*...*...*...*...*...*...*......*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*.......*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*..*...*...*...**.*...*...*...*...*...**..*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*.......*...*...*...*...*...**.*...*...*...*...*...*...*.*.*...*...*...*..*...*...*...*...*...*...*...*...*...**..*...*...*...*...*...*...*...*...*...*...*...*...*...*.......*...*...*...*...*...*...*...**..*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*..**...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*.*.**.**...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*.......*...*...*...*.*.*.*.*...*...*...*...*...*...*...*...*...*...**..*...*...*.*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*...*.......*...*...*..*...*...*...*...*.*.*...*...*test: This test is taking too long. Please retry, and if still failing send the log to me
CallStack (from HasCallStack):
  error, called at tests/Deadlocks.hs:137:13 in main:Deadlocks
```

**Comment by jberryman** [comment]
Thanks, and sorry for the delay getting to this. I appreciate the help and nudge. Going to close this for now since it probably needs CPP and I don't want all the whitespace changes, but I'll credit you. Thanks for doing the leg work. 

Will investigate the tests, may reach out to you if I can't reproduce the failure. Hope to have a new release today or later this weekend.

File: tests/UnagiUnboxed.hs [hunk 2]
```diff
@@ -79,13 +80,11 @@ applyToAllPrim f = do
     f (minBound :: Word16)
     f (minBound :: Word32)
     f (minBound :: Word64)
-    f (P.nullAddr `P.plusAddr` 1024 :: P.Addr)
+    f (nullPtr `advancePtr` 1024 :: (Ptr Word))
```
Comment (jberryman) [inline tests/UnagiUnboxed.hs]: not equivalent

**Pull request closed without merging**